datacenter/dtr/2.3/guides/admin/configure/use-a-web-proxy.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Use a web proxy
+description: Learn how to configure Docker Content Trust to use a web proxy to
+  reach external services.
+keywords: dtr, configure, http, proxy
+---
+
+Docker Trusted Registry makes outgoing connections to check for new versions,
+automatically renew its license, and update its vulnerability database.
+If DTR can't access the internet, then you'll have to manually apply updates.
+
+One option to keep your environment secure while still allowing DTR access to
+the internet is to use a web proxy. If you have an HTTP or HTTPS proxy, you
+can configure DTR to use it. To avoid downtime you should do this configuration
+outside business peak hours.
+
+As an administrator, log into a node where DTR is deployed, and run:
+
+```
+docker run -it --rm \
+  {{ page.dtr_org }}/{{ page.dtr_repo }}:{{ page.dtr_version }} reconfigure \
+  --http-proxy http://<domain>:<port> \
```

Comment:
I'm not too sure about the details of http vs https proxies. I think generally you set only one or the other?

Reply:
I also don't know a lot about proxies, but I'm seeing some blog posts telling you do configure your environment like this

```
 export http_proxy=http://<ip-address>
 export https_proxy=$http_proxy
```

So it seems that it's kind of a best practice to define both.